src/Sarif.Converters/FortifyConverter.cs

```diff
@@ -92,7 +92,13 @@ public static Result ConvertFortifyIssueToSarifIssue(FortifyIssue fortify)
         {
             var result = new Result();
             result.RuleId = fortify.Category;
-            result.ToolFingerprintContribution = fortify.InstanceId;
+
+            if (!string.IsNullOrWhiteSpace(fortify.InstanceId))
+            {
+                result.ToolFingerprintContributions = new Dictionary<string, string>();
```

Comment:
Are we sure that `result.ToolFingerprintContributions` doesn't already have a value at this point? And are we sure that if it does exist, it doesn't already contain an entry for `InstanceId`?

Reply:
I am not sure. I will add some checks.

Reply:
Actually yes we do know, because the parameterless Result constructor is being called, and the fingerprints property isn't initialized. Still, I think doing the checks is a good idea.